config/config.go

```diff
@@ -614,6 +620,7 @@ func DefaultInstrumentationConfig() *InstrumentationConfig {
 	return &InstrumentationConfig{
 		Prometheus:           false,
 		PrometheusListenAddr: ":26660",
+		MaxOpenConnections:   1,
```

Comment:
This default should be higher, given that a typical HA setup for Prometheus is identical active/active servers.

Reply:
Right. Thanks!